src/Components/Admin/navbar.js

```diff
@@ -1,7 +1,10 @@
-import React, { useContext } from "react";
+import React, { Fragment, useContext } from "react";
```

Comment:
React tip: you don't need to use `Fragment` and import it. You can instead use `<>` and `</>` instead. So, you code would look like
```
<>
    <div>...</div>
    <div>...</div>
</>
```
rather than:
```
<Fragment>
    <div>...</div>
    <div>...</div>
</Frament>
```